frontend/src/lib/components/DateFilter/DateFilter.tsx

```diff
@@ -115,7 +147,17 @@ export function DateFilter({
                     )
 
                     return (
-                        <Tooltip key={key} title={makeLabel ? makeLabel(dateValue) : undefined}>
+                        <Tooltip
+                            key={key}
+                            title={
+                                makeLabel
+                                    ? makeLabel(
+                                          dateValue,
+                                          dateFilterToText(values[0], undefined, '', [], false, 'MMMM D, YYYY', true)
```

Comment:
Nit: `const whatThisIs = ...;` above and then `makeLabel(dateValue, whatThisIs)` here